compose-postgres.yaml

```diff
@@ -0,0 +1,66 @@
+# Comments are provided throughout this file to help you get started.
+# If you need more help, visit the Docker compose reference guide at
+# https://docs.docker.com/compose/compose-file/
+
+# Here the instructions define your application as a service called "server".
+# This service is built from the Dockerfile in the current directory.
+# You can add other services your application may depend on here, such as a
+# database or a cache. For examples, see the Awesome Compose repository:
+# https://github.com/docker/awesome-compose
+services:
+  db:
+    image: postgres:16.1-alpine
+    restart: always
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+    ports:
+      - "5432:5432"
+    volumes:
```

Comment:
do we need persistency? I think we could skip the volume part

Reply:
you must have persistency with multi instance, that's how state is shared

Reply:
technically we share the same postgres, so you don't have to have persistency

but it is normally expected from postgres to be persistence 
so I prefer to have it